cumulus/constants.py

```diff
@@ -18,14 +18,20 @@
 ###############################################################################
 
 
+from enum import IntEnum
+
+
 class ClusterType:
     EC2 = 'ec2'
+    ANSIBLE = 'ansible'
     TRADITIONAL = 'trad'
```

Comment:
I agree,  having a type of 'ansible' is just a stop gap until we can get feature parity with starcluster then (hopefully) pull out starcluster and replace the naming convention